CHANGELOG.md

```diff
@@ -170,15 +170,15 @@ Changelog
 ### Changes
 
 - Use inclusive language
-- `slave` is deprecated in favor of `port`
-- `master` is deprecated in favor of `controller`
+- `slave` is deprecated in favor of `port` <!--- wokeignore:rule=slave -->
```

Comment:
The subject line of this PR may need to be properly adjusted. Because the changes are about workarounding the woke non-inclusive language failure with the `wokeignore: rule=*` comment.